examples/strip-locs/src/Main.hs

```diff
@@ -47,9 +47,9 @@ import "ghc-lib-parser" Platform
 #endif
 
 #if defined (GHC_MASTER) || defined (GHC_8101)
-import "ghc-lib" GHC.Hs.Dump
+import "ghc-lib-parser" GHC.Hs.Dump
```

Comment:
Can we change the cabal file of this example to no longer depend on `ghc-lib`?

Reply:
Ah yes!